apps/ejabberd/src/mongoose_lib.erl

```diff
@@ -0,0 +1,11 @@
+-module(mongoose_lib).
```

Comment:
The binary looks like "123rest". And I need to parse "rest" as a next step. I don't know where to put this function, but last time I added functions to mod_mam_utils, other developers wanted to use the functions inside other modules too.